tests/rptest/services/redpanda.py

```diff
@@ -154,6 +154,9 @@
     "raft - .*recovery append entries error.*client_request_timeout"
 ]
 
+# Path to the LSAN suppressions file
```

Comment:
nit: a comment that indicates the purpose of this file could be useful to first time readers